ruby-challenges.rb

```diff
@@ -3,12 +3,26 @@
 
 # --------------------1) Create a method that takes in an array of words and a single letter and returns an array of all the words containing that particular letter.
 
-beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+# letter_o = 'o'
+# # Expected output: ['coffee', 'soda water']
+# letter_t = 't'
+# # Expected output: ['tea', 'water', 'soda water']
 
-letter_o = 'o'
-# Expected output: ['coffee', 'soda water']
-letter_t = 't'
-# Expected output: ['tea', 'water', 'soda water']
+# beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+
+# def excluding ()
+#     if letter_o == 'o'
+#     puts ['coffee', 'soda water']
+#     elsif letter_t == 't'
+#         puts ['tea', 'water', 'soda water']
+#     end
+# end
+
+
+
+p beverages_array.excluding()
+
+# --------------------------------------private method `excluding' called for ["coffee", "tea", "juice", "water", "soda water"]:Array (NoMethodError), so I'm confused even though I'm not getting the expected output its saying 'no method error'.
```

Comment:
Hey Venessa - this has the same issue that the question in js had =>>. takes in an array of words and a single letter